core/src/main/java/com/linecorp/armeria/common/HttpHeaderGetters.java

```diff
@@ -49,6 +52,27 @@
     @Nullable
     MediaType contentType();
 
+    /**
+     * Returns a list of {@link LanguageRange}s that are specified in {@link HttpHeaderNames#ACCEPT_LANGUAGE}
+     * in the order of client-side preferences. If the client does not send the header, this will contain only a
+     * wild card {@link LanguageRange}.
+     */
+    List<LanguageRange> acceptLanguages();
```

Comment:
ditto
Shouldn't we put this into `RequestHeaderGetters`?

Reply:
Done